lib/ws.js

```diff
@@ -69,6 +72,35 @@ class BinanceWS {
         return data;
     }
 
+    _clearUserDataInterval() {
+        this.userDataRefreshIntervalId &&
+            clearInterval(this.userDataRefreshIntervalId);
+        this.userDataRefreshIntervalId = false;
+    }
+
+    _sendUserDataKeepAlive(binanceRest, response) {
+        return binanceRest.keepAliveUserDataStream(response).catch(e => {
+            this.userDataRefreshIntervalId.failCount++;
+            const msg =
+                'Failed requesting keepAliveUserDataStream for onUserData listener';
+            if (e && e.code === -1125) {
+                console.error(
+                    new Date(),
+                    msg + ' listen key expired - clearing keepAlive interval',
```

Comment:
Maybe some consistency here with the other call to `console.error`?  Do them as separate arguments rather than concatenating?